TravelPoints-UI/src/app/components/navigation-bar/navigation-bar.component.ts

```diff
@@ -46,9 +52,10 @@ export class NavigationBarComponent implements OnInit{
         alert("LogOut successful!")
         localStorage.clear()
         this.token = null
```

Comment:
make the token of type string | undefined instead of string | null to respect the typescript coding guidelines